docs/installation/configuration.mdx

```diff
@@ -0,0 +1,180 @@
+---
+sidebar_position: 10
+---
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Konfiguration
+
+:::note
+Diese Anleitung richtet sich an Benutzer, die evcc als Home Assistant Addon oder als Docker Container verwenden und deshalb
+den Konfigurationsassistent nicht verwenden können.
+Die Anleitung enthält aber eventuell nützliche Tipps, für Personen die ihre Konfiguration, nach der Verwendung des Konfigurationsassistenten noch anpassen müssen.
+:::
+
+Es wird im folgenden davon ausgegangen dass du eine leere Konfigurationsdatei _evcc.yaml_ im korrekten Verzeichnis erstellt hast.
+Die Anleitung findest du in der jeweiligen Installationsanleitung für Docker oder Home Assistant.
+
+```sh
+- config_file Docker:  `/etc/evcc.yaml`
+- config_file Home Assistant: `/config/evcc.yaml`
+```
+
+Kopiere den Inhalt der folgenden minimalen Template Vorlage in deine soeben erstellte Datei _evcc.yaml_.
+Die Vorlage erstellt eine Standardkonfiguration mit statischen Demo-Entitäten.
+
+```yaml
+## Configuration template with demo entities (meters, chargers) ##
+network:
+  # schema is the HTTP schema
+  # setting to `https` does not enable https, it only changes the way URLs are generated
+  schema: http
+  # host is the hostname or IP address
+  # if the host name contains a `.local` suffix, the name will be announced on MDNS
+  # docker: MDNS announcements don't work. host must be set to the docker host's name.
+  host: evcc.local
+  # port is the listening port for UI and api
+  # evcc will listen on all available interfaces
+  port: 7070
+
+interval: 30s # control cycle interval. Interval <30s can lead to unexpected behavior, see https://docs.evcc.io/docs/reference/configuration/interval
+
+# database configuration for persisting charge sessions and settings
+# database:
+#   type: sqlite
+#   dsn: <path-to-db-file>
+
+# sponsor token enables optional features (request at https://sponsor.evcc.io)
+# sponsortoken:
+
+# telemetry enables aggregated statistics
+#
+# Telemetry allows collecting usage data (grid and green energy, charge power).
+# It can be enabled in the configuration section of the UI.
+
+
+# log settings
+log: info
+levels:
+  site: debug
+  lp-1: debug
+  lp-2: debug
+  cache: error
+  db: error
+
+# modbus proxy for allowing external programs to reuse the evcc modbus connection
+# each entry will start a proxy instance at the given port speaking Modbus TCP and
+# relaying to the given modbus downstream device (either TCP or RTU, RS485 or TCP)
+modbusproxy:
+  #  - port: 5200
+  #    uri: solar-edge:502
+  #    # rtu: true
+  #    # readonly: true # use `deny` to raise modbus errors
```

Comment:
```suggestion
```

Ich würd die Optionen bis hier hin alle rausnehmen. Das ist wieder viel kopierter Text, denn wir mühselig in-sync halten müssen. Hier gehts ja nicht um nen Referenz sondern um ne Mini-Config. Gleiches betrifft natürlich die englische Übersetzung.

Reply:
Im Prinzip was es schon als Mini Template gedacht, um mit etwas beginnen zu können was ohne Fehlermeldung läuft. Wenn es nur eine Demo ist mit was soll ein Benutzer dann anfangen seine Konf zu erstellen?  Ich würde immerhin die Network und Log settings drin lassen.